backend/uclapi/oauth/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from .app_helpers import generate_user_token
+
+class OAuthScope(models.Model):
+    # We should really have a primary key for the OneToOneField
+    id = models.AutoField(primary_key=True)
```

Comment:
Interestingly, you can solve most issues with id fields (from the looks of things) by setting the values to `None`; Django will then assign an ID for you. That's probably how we could have solved that issue back then.